_data/apis.yml

```diff
@@ -531,6 +816,7 @@
 
 - name: Watson Language Translator
   id: watson
+  company: ibm-watson
```

Comment:
The company should be just IBM, no?

Reply:
Sure. The thing is that IBM Watson is a suborganization of IBM that was launched in 2014. IBM as a company was launched in 1911. They have different Crunchbase pages and everything.

I guess I find it a bit confusing since companies that offer different IT services generally did not start working on MT when they launched, but companies that just work on MT did. IDK.